src/calendar-wc.styles.ts

```diff
@@ -134,7 +134,7 @@ const CalendarStyles = css`
     font-size: 0.75rem;
     width: 14.2857%;
     height: 26.4px;
-    display: flex;
+      display: flex;
```

Comment:
@oluwatobi1 for formatting purpose, can you revert this line change back to the previous, so there is no leading space